src/lib.ts

```diff
@@ -24,4 +24,7 @@ export {
   fetchStatus,
   kind,
   containers,
+  writeEvent,
```

Comment:
we chatted about importing these directly from `pepr/sdk`. Not sure where we fell on this. Not sure that I have a strong opinion either

Reply:
would love to get a test around these in excellent-examples too before we merge this